### PR TITLE
chore: adaptations for nightly-2024-06-27

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 1]
```diff
@@ -124,10 +124,12 @@ lemma exists_isTuranMaximal :
 
 end Defs
 
-namespace IsTuranMaximal
+section Forward
```
Comment (semorrison) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:128]: I suspect this, and the corresponding `end Forward` at the end, must be a bad merge.

File: Mathlib/Computability/Primrec.lean [hunk 2]
```diff
@@ -5,8 +5,8 @@ Authors: Mario Carneiro
 -/
 import Mathlib.Logic.Equiv.List
 import Mathlib.Logic.Function.Iterate
-import Mathlib.Data.List.GetD
 import Mathlib.Algebra.Order.Ring.Nat
+import Mathlib.Data.List.GetD
```
Comment (semorrison) [inline Mathlib/Computability/Primrec.lean:9]: Again, merge problems, let's reverse these to minimize the diff.

**Comment by semorrison** [comment]
bors d+

File: Mathlib/Computability/Primrec.lean [hunk 3]
```diff
@@ -3,10 +3,10 @@ Copyright (c) 2018 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
+import Mathlib.Algebra.Order.Ring.Nat
```
Comment (jcommelin) [inline Mathlib/Computability/Primrec.lean:6]: I ended up alphabetically sorting these imports.

**Pull request merged**